### PR TITLE
Extend Kusama coverage on several more docs

### PR DESCRIPTION
**Pull request by alfarok**
Part of the #4110 initiative.

This PR covers:
- [x] build-hackathon
- [x] build-oracle
- [x] build-smart-contracts
- [x] build-wallets
- [x] contributing
- [x] learn-availability
- [x] learn-consensus
- [x] learn-identity
- [x] learn-runtime-upgrades

### CODE REVIEW

File: docs/build/build-smart-contracts.md [hunk 1]
```diff
@@ -240,21 +243,15 @@ Try deploying a confidential smart contract by following their
...
-Developers may be interested in joining the
-[Polkadot Beginners Lounge](https://matrix.to/#/#polkadotnoobs:matrix.org) or
-[Substrate Technical](https://area51.stackexchange.com/proposals/126136/substrate) to ask their questions.
-As always, keep up to date with Polkadot and Kusama by following the
-[social channels](../general/community.md). :kusama }}
+If you have interesting ideas for smart contracts on
+{{ polkadot: Polkadot, :polkadot }}{{ kusama: Kusama, :kusama }} feel free to drop into the
+{{ polkadot: [Polkadot Watercooler](https://matrix.to/#/#polkadot-watercooler:matrix.org) :polkadot }}
+{{ [Kusama Watercooler](https://matrix.to/#/#kusama-watercooler:matrix.org) :kusama }} to talk about
+them. Developers may be interested in joining the
```
Comment (filippoweb3) [inline docs/build/build-smart-contracts.md]:
```suggestion
{{ kusama: [Kusama Watercooler](https://matrix.to/#/#kusama-watercooler:matrix.org) :kusama }} to talk about
them. Developers may be interested in joining the
```

File: docs/learn/learn-consensus.md [hunk 2]
```diff
@@ -51,9 +51,10 @@ economies of scale. These pools are often off-chain.
 A way to alleviate this is to implement pool formation on-chain and allow token holders to vote
 [with their stake] for validators to represent them.
```
Comment (filippoweb3) [inline docs/learn/learn-consensus.md]:
```suggestion
with their stake for validators to represent them.
```

**Review by filippoweb3 — APPROVED** [review]
lgtm, just a couple of tiny edits ;)

File: docs/build/build-smart-contracts.md (hunk 1 above)
Comment (alfarok) [reply docs/build/build-smart-contracts.md]: Good catch!

**Pull request merged**